### PR TITLE
CU-26etwa2 CI job to build a Docker image of composable-sandbox

### PR DESCRIPTION
**Pull request by andor0**

### CODE REVIEW

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-26etwa2 Create CI job to build a Docker image of composable-sandbox](https://app.clickup.com/t/26etwa2)

File: scripts/polkadot-launch/README.md [hunk 1]
```diff
@@ -79,6 +79,12 @@ Need to do to run 4 relay chain nodes, 2 Composable's collators and 2 Basilisk's
 	yarn composable_and_basilisk
 	```
 
+or you can use Docker image
+
+	```bash
+	docker run --rm -ti -u$(id -u):$(id -g) -p9944:9944 -p9988:9988 -p9998:9998 composablefi/composable-sandbox:latest
```
Comment (dzmitry-lahoda) [inline scripts/polkadot-launch/README.md:85]: may be remove copy paste from here and force people to use make (avoid copy paste)

**Review by dzmitry-lahoda — APPROVED** [review]

File: scripts/polkadot-launch/README.md (hunk 1 above)
Comment (andor0) [reply scripts/polkadot-launch/README.md:85]: Yes, but it is required to install cargo-make, Rust and etc.

Comment (dzmitry-lahoda) [reply scripts/polkadot-launch/README.md:85]: but make requires install make and docker docker. do you promise to support 2 versions of run command up to date i 2 places?

**Review by KaiserKarel — APPROVED** [review]

**Pull request merged**